### ISSUE TITLE
noVNC c6c8e5e51329001f778a72607bc68432b9d359f8 with GET host parameter set now uses the host as the WSS endpoint

### ISSUE THREAD

**Issue by ichdasich** [novnc/noVNC#1890]
**Describe the bug**
noVNC c6c8e5e51329001f778a72607bc68432b9d359f8 with GET host parameter set now uses the host as the WSS endpoint.

Example:

https://application.example.com/novnc/?autoconnect=1&reconnect=0&host=api.example.com&encrypt=1&path=/websockify&password=passwd&shared=1&resize=scale

The browser tries to open a websocket to application.example.com:

```
WebSocket connection to 'wss://application.example.com/websockify' failed
```

I'd guess that this is a change in default behavior via

`074fa1a40f6e3f65bd61109de3f404796e266524`

or 

`96c76f7709037956e760ae5f58dbde1bc1308bba`

**To Reproduce**
1. Setup noVNC `c6c8e5e51329001f778a72607bc68432b9d359f8` on application.example.com
2. Run a websocket on api.example.com/websockify
3. Call noVNC with a browser as: https://application.example.com/novnc/?autoconnect=1&reconnect=0&host=api.example.com&encrypt=1&path=/websockify&password=passwd&shared=1&resize=scale
4. See `WebSocket connection to 'wss://application.example.com/websockify' failed` in console.log

**Expected behavior**
Backward compatibility and/or updated documentation on new way to set the websocket host, i.e., the `host` GET still being honored.

**Client (please complete the following information):**
 - OS:  Gentoo Linux
 - Browser: Google Chrom
 - Browser version: Version 126.0.6478.182 (Official Build) (64-bit)

**Server (please complete the following information):**
 - noVNC version: c6c8e5e51329001f778a72607bc68432b9d359f8
 - VNC server: QEMU (No change since last known good configuration)
 - WebSocket proxy: websockify (No change since last known good configuration, no longer receiving requests)

**Additional Context:**
Reverting to `06f14a5cd3c188c63ad2060b7532a914c0d8f7f6` fixes the issue.

**Comment by CendioOssman** [comment]
Indeed. Let me have a look at what went wrong.

**Comment by CendioOssman** [comment]
Query parameters were ovelooked in 96c76f7. Should be fixed in bbb6a5b938d4dd85a99ca78718b09dc08bc9a81d.

Thanks for letting us know!

**Comment by ichdasich** [comment]
Just tested, works; Thanks!
